test/extern/bradc/classes/extern_subclasses.good

```diff
@@ -1,2 +1 @@
-extern_subclasses.chpl:5: error: External types do not currently support inheritance
-extern_subclasses.chpl:13: error: External types do not currently support inheritance
+extern_subclasses.chpl:5: syntax error: near 'C2'
```

Comment:
Ugh it was nicer before.
Not a show stopper though. Would be nice to add an "error message" future.

Reply:
`extern class` has been deprecated more than 1 release. If `extern class` is not part of the language, I don't see any particular reason why we need to expend energy making a better error message for it.